net/nimble/host/mesh/src/beacon.c

```diff
@@ -169,9 +169,8 @@ static int unprovisioned_beacon_send(void)
 	net_buf_add_u8(buf, BEACON_TYPE_UNPROVISIONED);
 	net_buf_add_mem(buf, prov->uuid, 16);
 
-	if (prov->uri) {
+	if (prov->uri && bt_mesh_s1(prov->uri, uri_hash) == 0) {
```

Comment:
Commit message points to Coverity issue. Is this ours or Zephyr one? If Zephyr we should mentioned it or just skip this comment.